packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -232,6 +232,18 @@ export class GCFBootstrapper {
     }
   }
 
+  private static parseSignatureHeader(request: express.Request): string {
```

Comment:
In a comment for this function, explain why it returns 'unset'.  I would have expected it to return the empty string when a signature was not found in the request.

Reply:
Done